### PR TITLE
move /srv/webknossos to /webknossos in the dockerfiles

### PR DESCRIPTION
**Pull request by normanrz**
Moves the webKnossos files in the docker container from `/srv/webknossos` to `/webknossos`. This will make deployment on our infrastructure much easier, because we can mount `/srv` wholly and all symlinks etc will work internally.

Unfortunately, this brings a migration burden, where all mount-configs need to be changed. Would love to hear your feedback on that.

This PR also cleans up some unused and probably out-of-date scripts.

------
- [x] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [x] Updated [(unreleased) migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [x] Updated [documentation](../blob/master/docs) if applicable
- [x] Needs datastore update after deployment
- [x] Ready for review

### CODE REVIEW

**Comment by philippotto** [comment]
Sounds reasonable to me :+1:

**Review by fm3 — COMMENTED** [review]
I can’t really estimate how much effort the migration is, but seems reasonable to me too.

File: Dockerfile [hunk 1]
```diff
@@ -17,6 +17,9 @@ RUN addgroup --system --gid 999 webknossos \
   && chmod go+x bin/webknossos \
   && chmod go+w .
 
+RUN echo '#!/bin/bash\numask 002\n/bin/bash "$@"\n' > /docker-entrypoint.sh \
+  && chmod +x /docker-entrypoint.sh
```
[the remark this answers is not in the document]
Comment (jstriebel) [reply Dockerfile:21]: PS: You could also bake the `bin/webknossos` part into the entrypoint to keep it more backwards-compatible. Also you don't need another bash call, e.g.
```suggestion
RUN echo '#!/bin/bash\numask 002\nbin/webknossos "$@"\n' > /docker-entrypoint.sh \
  && chmod +x /docker-entrypoint.sh
```

File: webknossos-tracingstore/Dockerfile [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
-VOLUME /srv/webknossos-tracingstore/tracingData /tmp
+VOLUME /webknossos-tracingstore/tracingData /tmp
 
 COPY target/universal/stage .
 
 RUN chown -R webknossos . \
   && chmod go+x bin/webknossos-tracingstore \
   && chmod go+w .
 
+RUN echo '#!/bin/bash\numask 002\n/bin/bash "$@"\n' > /docker-entrypoint.sh \
```
Comment (jstriebel) [inline webknossos-tracingstore/Dockerfile:18]: Same as above.

File: webknossos-datastore/Dockerfile [hunk 3]
```diff
@@ -15,6 +15,9 @@ RUN chown -R webknossos . \
   && chmod go+x bin/webknossos-datastore \
   && chmod go+w .
 
+RUN echo '#!/bin/bash\numask 002\n/bin/bash "$@"\n' > /docker-entrypoint.sh \
+  && chmod +x /docker-entrypoint.sh
+
```
Comment (jstriebel) [inline webknossos-datastore/Dockerfile:20]: Same as above.

**Review by jstriebel — COMMENTED** [review]
Nice! I mostly added one suggestion to keep the `bin/webknossos` call part of the entrypoint, which allows to keep the current `command`s as they are.

> Unfortunately, this brings a migration burden, where all mount-configs need to be changed. Would love to hear your feedback on that.

I think that's fine :+1:

**Review by jstriebel — APPROVED** [review]
Nice, LGTM :+1:

**Pull request merged**